src/console-services.yml

```diff
@@ -191,50 +415,98 @@
+- command: sagemaker
+  description: 'SageMaker - Build, Train, and Deploy Machine Learning Models'
+  url: 'https://console.aws.amazon.com/sagemaker/home'
+
+- command: secretsmanager
+  description: 'Secrets Manager - Easily rotate, manage, and retrieve secrets throughout their lifecycle'
+  url: 'https://console.aws.amazon.com/secretsmanager/home'
+
+- command: securityhub
+  description: 'Security Hub - AWS’s security and compliance center'
+  url: 'https://console.aws.amazon.com/securityhub/home'
+
+- command: serverlessrepo
+  description: 'Serverless Application Repository - Assemble, deploy, and share serverless applications within teams or publicly'
+  url: 'https://console.aws.amazon.com/serverlessrepo/home'
+
+- command: servermigrationservice
+  description: 'Server Migration Service - Migrate on-premises servers to AWS'
+  url: 'https://console.aws.amazon.com/servermigrationservice/home'
+
+- command: servicecatalog
+  description: 'Service Catalog - allows organizations to manage approved catalogs of IT resources and make them available to employees via a personalized portal.'
+  url: 'https://console.aws.amazon.com/servicecatalog/home'
 
 - command: ses
-  description: 'SES - Email Sending and Receiving Service'
+  description: 'Simple Email Service - Email Sending and Receiving Service'
   url: 'https://console.aws.amazon.com/ses/home'
 
+- command: transfer
+  description: 'Transfer for SFTP - Fully-managed SFTP service for Amazon S3'
+  url: 'https://console.aws.amazon.com/transfer/home'
+
+- command: shield
+  description: 'WAF & Shield - Protects Against DDoS Attacks and Malicious Web Traffic'
+  url: 'https://console.aws.amazon.com/shield/home'
+
+- command: singlesignon
+  description: 'Single Sign-On - Manage Single Sign-On (SSO) Access to AWS Accounts and Business Applications'
+  url: 'https://console.aws.amazon.com/singlesignon/home'
+
+- command: snowball
+  description: 'Snowball - Large Scale Data Transport'
+  url: 'https://console.aws.amazon.com/snowball/home'
+
 - command: sns
-  description: 'SNS - Push Notification Service'
+  description: 'Simple Notification Service - SNS Pub/Sub, Mobile Push and SMS'
   url: 'https://console.aws.amazon.com/sns/home'
 
 - command: sqs
-  description: 'SQS - Message Queue Service'
+  description: 'Simple Queue Service - SQS Managed Message Queues'
   url: 'https://console.aws.amazon.com/sqs/home'
 
 - command: stepfunctions
-  description: 'Step Functions - Coordinate components of distributed applications'
+  description: 'Step Functions - Coordinate Distributed Applications'
   url: 'https://console.aws.amazon.com/states/home'
 
 - command: storagegateway
   description: 'Storage Gateway - Hybrid Storage Integration'
   url: 'https://console.aws.amazon.com/storagegateway/home'
 
-- command: swf
-  description: 'SWF - Workflow Service for Coordinating Application Components'
-  url: 'https://console.aws.amazon.com/swf/home'
```

Comment:
why was this removed?

Reply:
Again, probably because no new icon exists yet. Have used generic integration icon instead.